src/main.rs

```diff
@@ -168,6 +170,21 @@ fn run_tcp(cli_config: &CliConfig, dht_sk: SecretKey, tcp_onion: TcpOnion) -> im
         .map(|_| ())
         .map_err(|(e, _, _)| e);
 
+    let interval = Duration::from_secs(1);
```

Comment:
Can we rename current `run` method inside `ServerExt` to `run_connection` and create new `run` that incorporates pings?

Reply:
Good idea.

Reply:
`run` operates with only one connection so we will spam N pings for only 1 connection if we do that.

Reply:
I meant to create a new `run` method that runs all connections.

Reply:
Then we need to create a new Class. Right?